code/modules/projectiles/guns/ballistic.dm

```diff
@@ -611,7 +611,7 @@ GLOBAL_LIST_INIT(gun_saw_types, typecacheof(list(
 
 /obj/item/gun/ballistic/proc/guncleaning(mob/user, obj/item/A)
 	if(misfire_probability == 0)
-		to_chat(user, span_notice("[src] seems to be already clean of fouling."))
+		balloon_alert(user, "it's already clean!")
```

Comment:
```suggestion
		balloon_alert(user, "already clean...")
```

Reply:
this is wrong

Reply:
`!` is for failure, `...` is for pending